src/common/mmo.h

```diff
@@ -746,6 +746,7 @@ struct guild_member {
 	char name[NAME_LENGTH];
 	struct map_session_data *sd;
 	unsigned char modified;
+	uint32 login_date;
```

Comment:
i would rather if you keep the same name as in char `last_login`

Reply:
Done. Appveyor canceled the build of the VS 14, so error occurred